src/main/java/no/nav/tag/finnkandidatapi/kandidat/KandidatService.java

```diff
@@ -45,10 +52,19 @@
 
     private void oppdaterSistEndretFelter(Kandidat kandidat, Veileder innloggetVeileder) {
         kandidat.setSistEndretAv(innloggetVeileder.getNavIdent());
-        kandidat.setSistEndret(LocalDateTime.now());
+        kandidat.setSistEndret(dateProvider.now());
```

Comment:
💯 for `DateProvider`! Så mye bedre enn `LocalDateTime`, gjør koden mer testbar 👍